src/components/ControlPanel.vue

```diff
@@ -903,6 +956,72 @@ export default {
     showSnackbar (text) {
       this.$emit('showSnackbar', text)
     },
+    filterNodesByList (value, search, item, list) {
+      // Exclude not contained in selected list elements:
+      if (list !== null &&
+            list.length > 0 &&
+            !list.includes(value)
+      ) return false
+      return true
+    },
+    filterNodesStringCol (value, search, item, header) {
+      if (
+        header.search !== undefined &&
+        header.search !== null &&
+        typeof value === 'string') {
+          let i = value.toString().toLocaleLowerCase().indexOf(header.search.toLocaleLowerCase())
+          if (i === -1) return false
+      }
+      if (!this.filterNodesByList(value, search, item, header.filterInfo.list)) return false
+      return true
+    },
+    filterNodesNumberCol (value, search, item, header) {
+      let fi = header.filterInfo
+      if (
+        fi.min !== undefined &&
+        fi.min !== null &&
+        typeof value === 'number' &&
+        value < parseInt(fi.min)
+      ) return false
+      if (
+        fi.max !== undefined &&
+        fi.max !== null &&
+        typeof value === 'number' &&
+        value > parseInt(fi.max)
+      ) return false
+      if (!this.filterNodesByList(value, search, item, header.filterInfo.list)) return false
+      return true
+    },
+    filterNodesDateCol (value, search, item, header) {
+      return true
```

Comment:
I think this is an error

Reply:
Ah yes, that was the prepared code for extended date filtering - will remove it until it is done.